### ISSUE TITLE
Blog proposal - 2018

### ISSUE THREAD

**Issue by rub1e** [dwyl/blog#27]
**tl;dr** - with a bit of thought and effort, we can forget about the boring old company blog model; we can make a place to showcase the whole of dwyl; we can produce useful resources and insight for the community, while using the buzz we generate to grow interest in the wider organisation. #winwin
 
## Whys and wherefores

Lots of people have given this some thought over the months so **clearly there's some appetite**, but without someone taking the lead on it, *there are more ideas than plans*. 

Because of this, let's gather round the b-logfire stove and rekindle this discussion, starting with a proposal which people can debate without the pressure of having to start writing hanging over them.  

![The midnight society](https://vignette.wikia.nocookie.net/epicrapbattlesofhistory/images/0/03/Midnight_Society_watch.gif/revision/latest?cb=20151206183427)

## Opportunity / objectives

dwyl is a dev agency and a growing open source org full of people working for the cause of the light - **let's tell people about it!**

There are *conversations* going on that we could be a part of.

There are *people with the same values* that aren't aware of dwyl. They can't join us and collaborate if we don't **get the dwyl name and way of doing things out there**.

**dwyl has so many assets and resources to draw on** (some people call them "friends"), and has built up *serious goodwill* in the community. We can build on this and go even further with their help. 

* **Build dwyl's reputation for industry insight and thought leadership**:
  * *The dev shop in London* - content & materials to send to clients to showcase dwyl's unique approach and general awesomeness
  * *The wider open source organisation* - dwyl has over 400 members - this is a drop in the ocean compared to what could be achieved with some industry cachet and good PR

## Background research

No exaggeration to say there are a handful of companies in the world that use blogs in a really positive and consistently interesting way. 

From my desk research, it seems the norm is for companies to start a blog, write a few posts, forget about it for a few months, write a few more posts, then allow it to wither and die. **This looks worse than having no blog at all**.

It's also very obvious when a blog page only exists for SEO purposes, or just because they thought they should have one. **No identity, devoid of Purpose, no apparent reason for existing** other than "companies tend to have blogs and so should we".  

Additionally, even if companies do have a blog, no one talks about it - there is **no concept of organic discovery** with these blogs.

The upshot is that with a little effort the dwyl blog can easily be so much more - **dwyl has a clear identity and is wonderfully Purposeful**, we just need to find the best way to allow it to shine.

## Model 

![Free code camp](https://avatars0.githubusercontent.com/u/9892522?s=400&v=4)
> Let's go full Free Code Camp!

If you're not familiar with *Free Code Camp's* [blog](https://medium.freecodecamp.org/)... I can't even finish that sentence. Of course you are. 

FCC went from FAC-sized to super-sized in only three years. Much of that growth has been underpinned by their prolific blogging - because instead of limiting participation to just the org itself, they've encouraged submissions from their community.

So founder Quincy Larson has 50k Twitter followers and that's great, but you can see posts by random people with no following whose articles get *literally thousands of likes* and the shares that go with them. They have an **engaged audience and massive reach**. This is how FCC has established itself as a leading voice in the learning-to-code choir

So, **let's go full Free Code Camp!** 

*(Up to a point. I think they've allowed themselves to stray from being positively prolific into being a bit noisy. But still.)* 

## Proposal

*The dwyl Blog - focusing on dwyl's Open Source activities, on the work that dwyl does to make the world a better place, and on the vast team that's formed under the dwyl umbrella.* 

### Posts

A non-exhaustive list (add to it!)

* Repos / packages / tutorials created by dwyl that would be helpful to others (leveraging existing work)
* Reviews of new frameworks we're using
* Business lessons we've learned that are widely relevant (WHY it's important to have a Purpose, for example)
* Thought-pieces on the state of tech - ethics in tech, the open-source movement, etc.
* Tapping into conversations that are going on around us that we feel strongly about - e.g. diversity in tech, the open-source movement, etc.
* Marking occasions (e.g. Ada Lovelace day) and significant "moments"

### Approach

The only way this will work is if there's **an Editor (or editorial team) in charge** - otherwise it'll descend into something chaotic that doesn't reflect dwyl's identity or Purpose, and people will start treating it as an albatross around their necks. 

The Editor would:

* Maintain a list of topics / ideas / etc. that are relevant to dwyl
* Maintain a calendar of events that we can prepare for in advance
* (Proactively monitor social media for conversations to be part of)
* Maintain a list of potential contributors
* "Commission" articles from them (i.e. ask them to do it and then chase them when they forget)
* Help edit posts, take the pressure of contributors

This sounds like a lot of work but after a few hours' setup time, it *shouldn't even be an hour a week of ongoing commitment*.

### Schedule

No need to make a rod for our own backs from the beginning - we can *start slowly* with just core dwyl people (i.e. those who occasionally swing by no. 16), *just doing one post a fortnight*.

Then when we've got a bit of practice, and most importantly a bit of existing content, we can start asking our friends to contribute.

*Ideally we'd end up with a post every week*

### I've gone on too long, way past the word limit - please comment with your thoughts / volunteer to write the first post next week :stuck_out_tongue_winking_eye:

**Comment by Cleop** [comment]
Nice post @rub1e - well given that we're looking to launch Tachyons Boostrap this January, I think I'll be taking the plunge and writing our first post - see #16

**Comment by rub1e** [comment]
Great stuff @Cleop!

May I suggest, re #16, that you hold fire for a few days until there's been a bit more discussion of the above and an agreed strategy has emerged? That way we can answer your questions re tone, purpose, audience etc. *before* you start - I know how demoralising and counterproductive it can be to get feedback and have to make changes after you've already put in a load of work!

The other thing is that it would be good to get a bit of a pipeline ready in advance so we can give people a heads up - I hate the idea of leaving a month between posts because you're pestering people to write stuff 

Hopefully now that the new year has broken people will have the chance to put their tuppence-worth down about the merits of various aspects of the proposal 😄
